README.md

```diff
@@ -35,6 +35,8 @@ This table represents the supported components of AWS OTel Collector in 2020. Th
 |                                 | metricstransformprocessor     | newrelicexporter   |                        |
 |                                 |                               | sapmexporter       |                        |
 |                                 |                               | signalfxexporter   |                        |
+|                                 |                               | `awsprometheusremotewriteexporter`   |                        |
```

Comment:
is  "`" needed?

Reply:
I believe this counts as an in-house component, hence we labelled it the same way as the others

Reply:
From the Readme:
> The highlighted components below are developed by AWS in-house. The rest of the components in the table are the essential default components that AWS OTel Collector will support.